device/send.go

```diff
@@ -125,7 +129,8 @@ func (peer *Peer) SendHandshakeInitiation(isRetry bool) error {
 	peer.timersAnyAuthenticatedPacketTraversal()
 	peer.timersAnyAuthenticatedPacketSent()
 
-	err = peer.SendBuffer(packet)
+	// TODO(raggi): allocation could be avoided here
```

Comment:
Do not litter the source code with your name like this. (Applies elsewhere too.) This isn't a Google project.

Also, though - what's holding up taking care of this TODO now?

Reply:
Sorry, 10y at Google installs some habits.

cost/benefit - it requires adding new pooling one way or another, which isn't all that worthwhile right now. It may become substantially easier to do later, with either the new experimental arenas stuff that is coming, and/or a discussion we've yet to have around potentially reusing some fancy gear from gvisors netstack code. Ultimately the todo line was a marker to come back and evaluate if it had become worthwhile to try and do. I don't think it is right now, so I'll remove the TODO entirely.